datalake/src/main/resources/runtime/7.2.2/aws/medium_duty_ha.json

```diff
@@ -83,7 +63,7 @@
           "size": 50
         }
       },
-      "nodeCount": 2,
```

Comment:
I realized that IDBroker does have client-side load balancing. Can we leave this at 2 to enable that?

Reply:
This isn't something we have verified.  We can add this in later.

Reply:
Also RDC has issues when spinning up a datahub with 2 nodes of idbroker.